docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml

```diff
@@ -302,16 +305,16 @@ validation code: |
   if not(requestor.phone_number) and not(requestor.email):
     validation_error("You need to provide at least one contact method.",field="requestor.phone_number")
 ---
-sets:
-  - student.name.first
 id: student name
 question: |
   Who is this letter for?
 subquestion: |
   Write the name of the student whose records you are requesting.
 fields:
-  - code: |
-      student.name_fields()
+  - First name: student.name.first
```

Comment:
See better way to remove suffix

https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/question_library/names#optional-omit-suffix